src/resmom/start_exec.c

```diff
@@ -5488,276 +5476,3 @@ log_mom_portfw_msg(char *msg)
- *
- * @return 	int
- * @retval 	1	can"idate" matches master name
- * @retval 	0	not a match
- *
- */
-static int
-wchost_match(const char *can, const char *master)
-{
-	const char *pc;
-	const char *pm;
-
-	pc = can + strlen(can) - 1;
-	pm = master + strlen(master) - 1;
-	while((pc > can) && (pm > master)) {
-		if (tolower(*pc) != tolower(*pm))
-			return 0;
-		pc--;
-		pm--;
-	}
-
-	/* comparison of one or both reached the start of the string */
-	if (pm == master) {
-		if (*pm == '*')
-			return 1;
-		else if ((pc == can) && (tolower(*pc) == tolower(*pm)))
-			return 1;
-	}
-	return 0;
-}
-
-/* start localmod 009 */
-/**
- * @brief
- * 	Wild card suffix host name match.  Do a case insensitive compare since
- * 	hostnames are not case sensitive.
- *
- * @param can		stage request hostname
- * @param master	name from usecp list (may be wild carded at end)
- *
- * @return	int
- * @retval 	1	candidate matches master name
- * @retval 	0	not a match
- *
- */
-static int
-wcsuffix_host_match(const char *can, const char *master)
-{
-	int cmp_length;
-	int master_length = strlen(master);
-
-	if (master[master_length - 1] == '*') {
-		cmp_length = master_length - 1;
-	} else {
-		cmp_length = master_length;
-	}
-
-	if (strncasecmp(can, master, cmp_length) == 0)
-		return 1;
-
-	return 0;
-}
-/* end localmod 009 */
-
-/**
- * A path in windows is not case sensitive so do a define
- * to do the right compare.
- */
-#ifdef	WIN32
-#define	PATHCMP	strncasecmp
-#else
-#define	PATHCMP	strncmp
-#endif
-
-/**
- * @brief
- * 	Check a stage file request against the saved set of "usecp" paths.
- *
- * @param[in]	host	stage request hostname
- * @param[in]	oldpath	stage request path
- * @param[out]	newpath	pointer to "usecp" path
- *
- * @return	int
- * @retval 	1	file matched, newpath is updated
- * @retval 	0	no match, newpath is unchanged
- *
- */
-static int
-told_to_cp_nas(char *host, char *oldpath, char **newpath)
-{
-	int i;
-	int nh;
-	static char newp[MAXPATHLEN + 1];
-	extern struct cphosts *pcphosts;
-
-/* start localmod 009 */
-	int match_found = 0;
-	for (nh = 0; nh < cphosts_num; nh++) {
-		if (wchost_match(host, (pcphosts + nh)->cph_hosts) ||
-		    wcsuffix_host_match(host, (pcphosts + nh)->cph_hosts)) {
-			i = strlen((pcphosts + nh)->cph_from);
-			if (PATHCMP((pcphosts + nh)->cph_from, oldpath, i) == 0) {
-				if ((pcphosts + nh)->cph_exclude)
-					return 0;
-
-				match_found = 1;
-				(void) strcpy(newp, (pcphosts + nh)->cph_to);
-				(void) strcat(newp, oldpath + i);
-			}
-		}
-	}
-
-	if (match_found) {
-		*newpath = newp;
-	}
-
-	return match_found;
-/* end localmod 009 */
-}
-
-/**
- * @brief
- * 	Decide if the specified path is to a local or remote file.
- *
- * @param	path	pointer to stage path string
- *
- * @retval	1 if remote
- * @retval	0 if local
- *
- * @par Side Effects:
- * Updates the path pointer to just the path name if local.
- *
- */
-static int
-local_or_remote_nas(char **path)
```

Comment:
Why is this function removed? is checking against localhost or momhost not needed before calling told_to_cp?